pkg/resources/shared_database_acceptance_test.go

```diff
@@ -15,15 +18,30 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestAcc_CreateSharedDatabase_minimal(t *testing.T) {
-	t.Skip("To be unskipped in the next database pr")
-
+func TestAcc_CreateSharedDatabase_Basic(t *testing.T) {
 	id := acc.TestClient().Ids.RandomAccountObjectIdentifier()
 	comment := random.Comment()
 
 	newId := acc.TestClient().Ids.RandomAccountObjectIdentifier()
 	newComment := random.Comment()
 
+	var (
```

Comment:
Yeah, I was doing this as a second database and wanted to be "more strict" with calling outside of resource.Test, because it's called during unit/integration tests, so I'll use this approach in secondary databases. Not much of an improvement, but that's always something.

Reply:
oh, I see, I understand now